### PR TITLE
Fixed compile errors, updated MacOS docs

### PR DESCRIPTION
**Pull request by vgulkevic**
Fixes two compile errors introduced in recent commits and also updates the docs

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -1 +0,0 @@
-README.md
```
Comment (CryptoCoderz) [inline README]: This needs to remain.

**Review by CryptoCoderz — CHANGES_REQUESTED** [review]
Readme file needs to containt "README.md" which was deleted for some reason in this pull request.

**Comment by vgulkevic** [comment]
I returned the file back.

**Comment by vgulkevic** [comment]
For future reference, it is usually more professional to explain and not demand why something is required, expand your thoughts as someone making a PR can, perhaps, be unaware of something that you are aware of and etc. 
This might be a good read on how to review pull requests - https://www.elastic.co/guide/en/kibana/current/pr-review.html

**Comment by CryptoCoderz** [comment]
The reason I was "demanding" the change back is because if the change was made then anyone going to github would not be able to see or read the readme on the landing page of the project repository. This is something that is done with all github projects as they all link a "readme" file to the "readme.md"

**Comment by vgulkevic** [comment]
https://stackoverflow.com/questions/8655937/what-is-the-difference-between-readme-and-readme-md-in-github-projects#:~:text=md%20is%20markdown%20.,to%20generate%20github's%20html%20summary.

**Comment by CryptoCoderz** [comment]
Right, and our readme.md has all the instructions and markdowns... So by deleting it you would have removed all the instructions....

**Comment by vgulkevic** [comment]
image
image
image

**Comment by CryptoCoderz** [comment]
You no longer have to use this repository, please use the main branch. It will offer you the freedom to make whatever changes you see fit.

**Pull request closed without merging**